src/cdomains/listDomain.ml

```diff
@@ -2,6 +2,8 @@ open Cil
 
 module GU = Goblintutil
 
```

Comment:
For domains as generic as lists, I don't think it's an issue if there are unused functions. These form the future building blocks of other domains, so I'd argue we keep these.

Reply:
Ok, Github on the phone is crappy and posts things twice :/